src/utils/render.js

```diff
@@ -1,6 +1,6 @@
 import AbstractClass from "../view/abstract-class";
 
-export const renderPosition = {
+export const RENDER_POSITION = {
```

Comment:
```suggestion
export const RenderPosition = {
```
https://up.htmlacademy.ru/ecmascript/12/criteries#b7